lib/galaxy/tools/deps/resolvers/default_conda_mapping.yml

```diff
@@ -2,4 +2,46 @@
   to: r-base
 - from: blast+
   to: blast
+- from:
+    name: samtools
+    unversioned: true
+  to:
+    name: samtools
+    version: 1.3.1
+- from:
+    name: ucsc_tools
+    unversioned: true
+  to:
+    name: ucsc_tools
+    version: 332
```

Comment:
I guess this is stuck in https://github.com/galaxyproject/conda-iuc/pull/9/files, I have closed the bioconda PR that was going to add this.